doc/format-directory.md

```diff
@@ -42,12 +42,29 @@ in use by any open LSM handle in the session.
  * `${session}/active/${n}.index`
  * `${session}/active/${n}.checksums`
 
-The LSM runs are numbered using a session-wide counter that is initialised from
-0 each time a session is opened.
+In addition, there are files related to the write buffer rather than runs. In
+particular for storing blobs associated with entries in the write buffer.
+
+ * `${session}/active/${n}.wbblobs`
+
+The LSM runs (and write buffer blob files) are numbered using a session-wide
+counter that is initialised from 0 each time a session is opened.
 
 Note that there are no metadata files in the active directory (as there are for
 snapshots). This is because the metadata is held in memory for open LSM handles.
 
+# Write buffer blobs files
+
+The `${n}.wbblobs` file format is the same as the `${n}.blobs` format used for
+runs: the concatenation of all the blobs written out so far.
+
+One difference is that these files are shared between open tables created using
+the `duplicate` operation, and so can contain the blobs inserted via many table
+handles.
```

Comment:
Technically, blob files for runs are also shared on `duplicate`, but it's different in the sense that many tables might write to the same write buffer blob file, whereas the run blob files are immutable

Reply:
I've revised the wording.